tests/resources/properties/release.properties

```diff
@@ -0,0 +1,31 @@
+spring.batch.job.names=CREATE_INCREMENTAL_ACCESSION_RELEASE_JOB
+parameters.assemblyAccession=GCA_009858895.3
+parameters.assemblyReportUrl=file:{assembly_report_url}
+parameters.chunkSize=1000
+parameters.contigNaming=INSDC
+parameters.fasta={fasta_file}
+parameters.forceRestart=true
+parameters.outputFolder={processing_folder}
+#parameters.accessionedVcf=
```

Comment:
```suggestion
```
(or, add a comment that you don't need this because you're passing it in the nextflow)